tables/lg-ug-g1.utb

```diff
@@ -0,0 +1,58 @@
+# liblouis: Luganda Grade 1 Braille
+#
+#-display-name: Luganda Grade 1
+#-index-name: Luganda Grade 1
```

Comment:
This new table should be added to the extra/generate-display-names/display-names file. Running `make` in that directory will then allow to generate the correct "display-name" and "index-name" tags.

Reply:
Done